### PR TITLE
Add Remote Garage

### PR DESCRIPTION
**Pull request by harsh-webk22**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

File: company-profiles/remotegarage.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Docker, Kubernetes , React , Golang , GRPC, JavaScript.
+
+## Office locations
+
+The main offices are located in Banglore.
+
+## How to apply
+
+Visit our [Contact Page](http://www.remotegarage.club/?page_id=419/)
```
Comment (evalsocket) [inline company-profiles/remotegarage.md:33]: https://angel.co/company/remotegarage

File: company-profiles/remotegarage.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+We are a community of gophers, who do programming not because they expect to get paid or get adulation by public, But because it is fun to program.
+
+
+## Company size
+
+10+ employees (as of Aug 2020).
+
+## Remote status
+
+We highly believe and focus on providing remote employement.
```
Comment (evalsocket) [inline company-profiles/remotegarage.md:17]: Bringing Traditional Education to the way it was. Education should be free.	

Remote Garage works upon the concept of gurudakshna which allows anyone to learn the most demanding Technologies and prepares you to work with the opensource community across the globe with zero Investment.

File: README.md [hunk 3]
```diff
@@ -372,6 +372,7 @@ Name | Website | Region
 [RedMonk](/company-profiles/redmonk.md) ⚠️️ | http://redmonk.com/ |
 [Redox](/company-profiles/redox.md) | https://www.redoxengine.com/ | USA
 [reinteractive](/company-profiles/reinteractive.md) ⚠️️ | https://reinteractive.net/ |
+[Remote Garage](/company-profiles/remotegarage.md) | http://www.remotegarage.club/ | India
```
Comment (evalsocket) [inline README.md:375]: use remotegarage as name

File: company-profiles/remotegarage.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+
+# Remote Garage
```
Comment (evalsocket) [inline company-profiles/remotegarage.md:2]: use name Remotegarage

**Comment by auto-nylen** [comment]
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

**Comment by harsh-webk22** [comment]
@nylen please look into it.

**Comment by auto-nylen** [comment]
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

**Comment by auto-nylen** [comment]
Hi @nylen, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit 4463970). :heavy_check_mark:

**Comment by nylen** [comment]
Hi @harsh-webk22, generally the capitalization and spacing of the company name should match the way it is displayed on the official website, so I've updated this to "Remote Garage" everywhere.

Please take a look at the changes in d862d59 to improve the clarity of language and let me know if you have any questions or suggestions, or if I got something wrong there.

I also have a couple of other comments that I will leave inline.

File: company-profiles/remote-garage.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# Remote Garage
+
+## Company blurb
+
+[Remote Garage](http://www.remotegarage.club/) is on a mission to bring your best potential regardless of what was left undiscovered, by becoming the first school in India to launch a new career. Sound like a worthy endeavor? Come join us!
```
Comment (nylen) [inline company-profiles/remote-garage.md:5]:
```suggestion
[Remote Garage](http://www.remotegarage.club/) is on a mission to bring out your full potential as a full-stack Go programmer. Sound like a worthy endeavor? Come join us!
```

This change, or something like that, is a much clearer way to state the purpose of Remote Garage (I am guessing a bit at the "full-stack Go programmer" part). You certainly don't have to accept this specific suggestion but there are a couple of problems with the grammar and precision of the current wording:

>on a mission to bring your best potential
"bring out your full potential" is a more natural/correct way to say this.

>regardless of what was left undiscovered,
what would be left "undiscovered"? by whom?

>by becoming the first school in India to launch a new career
which new career? can you prove that you are the first in all of India?

**Review by dougaitken — APPROVED** [review]

**Review by dougaitken — APPROVED** [review]

**Review by dougaitken — APPROVED** [review]

**Pull request merged**